src/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,38 @@
+import { NextApiHandler } from 'next'
+import NextAuth from 'next-auth'
+import Providers from 'next-auth/providers'
+import prismaClient from '../../../lib/prisma'
+
+const providers = {
+  providers : [
+    Providers.Credentials({
+      name: 'Credentials',
+      credentials: {},
+      authorize: async (credentials) => {
+        const user = await prismaClient.user.findMany({
+          where: {
+            email: credentials.email,
+            password: credentials.password
+          }
+        }).catch(err => err)
+        // TODO: Send useful error message
+        return !user.code && user.length ? user[0] : new Error('The email or password is incorrect.')
```

Comment:
Do you think this variable will make your code more legible for other members?

```suggestion
        const isUserAuthorized = !user.code && user.length

        return isUserAuthorized ? user[0] : new Error('The email or password is incorrect.')
```